### PR TITLE
Fix table sorting

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
This PR fixes the table sorting, that broke with the vuetify 2.2 upgrade

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user

```

### CODE REVIEW

File: frontend/src/store/modules/shoots.js [hunk 1]
```diff
@@ -455,8 +455,10 @@ const shoots = (state) => {
 }
 
 const setSortedItems = (state, rootState) => {
-  const sortBy = get(state, 'sortParams.sortBy')
-  const descending = get(state, 'sortParams.descending', false) ? 'desc' : 'asc'
+  const sortBy = head(get(state, 'sortParams.sortBy'))
+  const sortDesc = get(state, 'sortParams.sortDesc', [false])
+  const descending = head(sortDesc) ? 'desc' : 'asc'
+  console.log(sortBy, descending)
```
Comment (holgerkoser) [inline frontend/src/store/modules/shoots.js:461]: remove `console.log`

Comment (petersutter) [reply frontend/src/store/modules/shoots.js:461]: ✅

File: frontend/src/store/modules/shoots.js [hunk 2]
```diff
@@ -455,8 +455,9 @@ const shoots = (state) => {
 }
 
 const setSortedItems = (state, rootState) => {
-  const sortBy = get(state, 'sortParams.sortBy')
-  const descending = get(state, 'sortParams.descending', false) ? 'desc' : 'asc'
+  const sortBy = head(get(state, 'sortParams.sortBy'))
+  const sortDesc = get(state, 'sortParams.sortDesc', [false])
+  const descending = head(sortDesc) ? 'desc' : 'asc'
```
Comment (holgerkoser) [inline frontend/src/store/modules/shoots.js:460]: I know it has nothing to do with this PR. But it is completely confusion that a variable with the name `descending` could have a value `'asc'`? We should rename `descending` into `order` or `sortOrder`.

Comment (petersutter) [reply frontend/src/store/modules/shoots.js:460]: ✅

**Pull request merged**